test/components/views/messages/EncryptionEvent-test.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
No worries for these ones, but we should be writing all new unit tests with react testing library

Reply:
Migrated :heavy_check_mark: